### PR TITLE
Typecheck pedigree

### LINKED ISSUE #1831: typecheck pedigree.py

### PR DESCRIPTION
**Pull request by danking**
Merge #1868 first

Resolves #1831

### CODE REVIEW

File: python/hail/representation/pedigree.py [hunk 1]
```diff
@@ -241,26 +259,28 @@ def filter_to(self, samples):
...
-        
+
             Phenotype information is not preserved in the Pedigree data structure in Hail.
             Reading and writing a PLINK .fam file will result in loss of this information.
-            Use the key table method :py:meth:`~hail.KeyTable.import_fam` to manipulate this 
+            Use the key table method :py:meth:`~hail.KeyTable.import_fam` to manipulate this
             information.
 
-        :param str path: output path
+        :param path: output path
```
Comment (tpoterba) [inline python/hail/representation/pedigree.py]: why this change?

Comment (danking) [inline python/hail/representation/pedigree.py]: I noticed this was different from the usual style we use, so I changed it to be in sync with what I saw elsewhere.

Comment (tpoterba) [inline python/hail/representation/pedigree.py]: If there's one type, we use this syntax:
```
:param str path: ...
```

If there are multiple possible types (or a :py:meth:...), we use this syntax:
```
:param path: ...
:type path: str or None
```
